### PR TITLE
[zh-cn] sync contribute/docs.md

### PR DESCRIPTION
**Pull request by my-git9**
content/zh-cn/docs/contribute/docs.md

### CODE REVIEW

File: content/zh-cn/docs/contribute/docs.md [hunk 1]
```diff
@@ -0,0 +1,383 @@
...
+---
+
+{{< note >}}
+<!--
+To learn more about contributing to Kubernetes in general, see the general
+[contributor documentation](https://www.kubernetes.dev/docs/) site.
+-->
+{{< /note >}}
+要了解有关为 Kubernetes 做出贡献的更多信息，请参阅
+[贡献者文档](https://www.kubernetes.dev/docs/)。
```
Comment (windsonsea) [inline content/zh-cn/docs/contribute/docs.md:51]:
```suggestion
要了解有关为 Kubernetes 做出贡献的更多信息，
请参阅[贡献者文档](https://www.kubernetes.dev/docs/)。
{{< /note >}}
```

**Review by windsonsea — COMMENTED** [review]
Seems those changes of #44686 shall be included in this PR.

**Comment by my-git9** [comment]
> Seems those changes of #44686 shall be included in this PR.

Updated

File: content/zh-cn/docs/contribute/docs.md [hunk 2]
```diff
@@ -0,0 +1,400 @@
...
+
+
+classDef grey fill:#dddddd,stroke:#ffffff,stroke-width:px,color:#000000, font-size:15px;
+classDef white fill:#ffffff,stroke:#000,stroke-width:px,color:#000,font-weight:bold
+classDef spacewhite fill:#ffffff,stroke:#fff,stroke-width:0px,color:#000
+class A,B,C,D,E,F,G,H,M,Q,N,O,P,V grey
+class S,T,U spacewhite
+class first,second,third white
+{{</ mermaid >}}
+-->
```
Comment (windsonsea) [inline content/zh-cn/docs/contribute/docs.md:134]: line 93-134 can be removed for the mermaid source.

File: content/zh-cn/docs/contribute/docs.md [hunk 3]
```diff
@@ -0,0 +1,400 @@
...
+     
+
+classDef grey fill:#dddddd,stroke:#ffffff,stroke-width:px,color:#000000, font-size:15px;
+classDef white fill:#ffffff,stroke:#000,stroke-width:px,color:#000,font-weight:bold
+classDef spacewhite fill:#ffffff,stroke:#fff,stroke-width:0px,color:#000
+class A,B,D,E,F,G grey
+class S,T spacewhite
+class first,second white
+{{</ mermaid >}}
+-->
```
Comment (windsonsea) [inline content/zh-cn/docs/contribute/docs.md:241]: Same issue, line 214-241 can be removed for the mermaid source.

**Review by windsonsea — COMMENTED** [review]
Mermaid source is not suitable to be commented out. It may cause the rendering in a mess. See current preview: https://deploy-preview-44659--kubernetes-io-main-staging.netlify.app/zh-cn/docs/contribute/docs/

**Review by ydFu — APPROVED** [review]
/approve

**Pull request merged**